src/account/UpgradeableModularAccount.sol

```diff
@@ -197,7 +230,7 @@ contract UpgradeableModularAccount is
         }
 
         PostExecToRun[] memory postExecHooks =
-            _doPreExecHooks(_storage.selectorData[selector].executionHooks, data);
+            _doPreHooks(_storage.selectorData[selector].executionHooks, data, false);
```

Comment:
I think `executeFromPlugin shouldn't exist in this PR, right? The permissions were removed in #67 on your stack, and #65 on mine.

Reply:
not rebased on top of that yet!